client/geoNamesClient.js

```diff
@@ -0,0 +1,21 @@
+const Geonames = require('geonames.js');
+
+const COUNTRY_CODES = 'US, CA';
+const CITIES_POPULATION_QUERY = 'cities5000';
+
+module.exports = {
+	getCities: async (name) => {
+		const geonames = Geonames({
+			username: process.env.USERNAME || 'badal_busbud',
```

Comment:
👮🏻‍♂️Do not hardcode the username here. This should be in a config file example and provide the value in the readme